docs/quickstart.rst

```diff
@@ -61,69 +147,67 @@ instance had already existing in the destination directory, this command
 would have configured it to serve as a monitor.
 
 In our case, ``pg_autoctl create monitor`` creates a database called
-``pg_auto_failover``, installs the ``pgautofailover`` Postgres extension, and grants access
-to a new ``autoctl_node`` user.
+``pg_auto_failover``, installs the ``pgautofailover`` Postgres extension, and
+grants access to a new ``autoctl_node`` user.
 
```

Comment:
We are missing the `pg_autoctl monitor run` command, which now outputs some interesting bits of information, and also should be registered on the system startup scripts (systemd integration). See below for more on that topic and the `pg_autoctl show systemd` command. I believe we should include that topic in our tutorial... what do you think?